test/Microsoft.AspNet.Mvc.Core.Test/DefaultControllerFactoryTest.cs

```diff
@@ -185,8 +184,10 @@ public void CreateController_ThrowsIfPropertyCannotBeActivated()
 
             // Act and Assert
             var exception = Assert.Throws<InvalidOperationException>(() => factory.CreateController(context));
-            Assert.Equal("The property 'Service' on controller '" + typeof(ControllerThatCannotBeActivated) +
-                        "' cannot be activated.", exception.Message);
+            Assert.Equal(
+                $"Unable to resolve service for type '{typeof(TestService)}' while attempting to activate " +
+                $"'{typeof(ControllerThatCannotBeActivated)}'.", 
```

Comment:
Minor icky-ness. Could you change to be explicit and use `.FullName` while here please?